docs/modules/language-reference/pages/index.adoc

```diff
@@ -3179,6 +3180,157 @@ This section discusses language features that are generally more relevant to tem
+<2> With an explicit element type argument, this property's default value is equivalent to `new Listing<Gizmo> { default = new Gizmo { name = "doodad" } }`
+<3> With an explicit value type argument, this property's default value is equivalent to `new Mapping<String, Gizmo> { default = (_) -> new Gizmo { name = "doodad" } }`
+
+==== Explicit Parent
+
+Explicit amending is covered in detail by <<amending-objects, Amending Objects>>.
+Amending a property, entry, or value results in a new value with the same type as the parent.
+When amending a property that has not been previously declared, an empty `Dynamic` object is used as the parent.
+
+[source,{pkl}]
+----
+newProperty { // <1>
+  hello = "world"
+}
+amendedProperty = (newProperty) { // <2> 
+  hello = "there"
+}
+someMapping = new Mapping {
+  ["greetings"] { // <3>
+    hello = "everyone"
+  }
+}
+----
+<1> Amending a property.
+<2> Amending a value.
+<3> Amending a new `Mapping` entry.
+
+==== Explicitly Typed `new`
+
+Instantiating an object with `new <type>` results in a value that amends the specified type's default value.
+Notably, creating a `Listing` element or assigning a `Mapping` value with an explicitly typed `new` ignores a collection's `default` value.
+
+[source,{pkl}]
+----
+newProperty = new Dynamic { // <1>
+  hello = "world"
+}
+someListing = new Listing {
+  default {
+    isDefault = true
+  }
+  new Dynamic { // <2>
+    hello = "there"
+  }
+}
+someMapping = new Mapping {
+  default {
+    isDefault = true
+  }
+  ["greetings"] = new Dynamic { // <3>
+    hello = "everyone"
+  }
+}
+----
+<1> Assigning a `new` explicitly-typed value to a property.
+<2> Adding an `new` explicitly-typed `Listing` element.
+The value will not have property `isDefault = true` as the `default` property of the `Listing` is not used.
+<3> Assigning a `new` explicitly-typed value to a `Mapping` entry.
+The value will not have property `isDefault = true` as the `default` property of the `Mapping` is not used.
+
+==== Implicitly Typed `new`
+
+When using the implicitly typed `new` invocation, there is no explicit parent value to amend.
+In these cases, Pkl infers the amend operation's parent value based on context.
+In order of precedence:
+
+1. When assigning to a declared property, the property's default value is amended (<<amend-null, including `null`>>).
+2. Within a `Listing` or `Mapping`, the container's function-typed `default` property is applied to the corresponding index or key, respectively, to produce the value to be amended.
+3. In module or object body context, when no type information is otherwise available, an empty `Dynamic` object is amended.
+4. In other cases, type inference fails with the error message "Cannot tell which parent to amend".
+
+Currently, the type information of a <<methods,method>> parameter is not used for type inference, which may lead to surprising type inference errors.
+In this case, argument type information should be specified explicitly.
```

Comment:
"Currently" suggests that this will change in the future, but I'm not so sure that we can ever change this.

```suggestion
The type information of a <<methods,method>> parameter is not used for type inference.
In this case, argument type information should be specified explicitly.
```